src/app/stream-deck/stream-deck.ts

```diff
@@ -0,0 +1,54 @@
+import { DimItem } from 'app/inventory/item-types';
+import { DimStore } from 'app/inventory/store-types';
+import { Loadout } from 'app/loadout-drawer/loadout-types';
+import { ThunkResult } from 'app/store/types';
+import { LazyStreamDeck, StreamDeckState } from 'app/stream-deck/interfaces';
+import { DeferredPromise } from 'app/stream-deck/util/deferred';
+import { removeClientIdentifier, removeStreamDeckToken } from 'app/stream-deck/util/local-storage';
+
+export const lazyStreamDeck: LazyStreamDeck = {};
+
+// load core function
+const loadStreamDeckAsync = async () => {
+  if (!lazyStreamDeck.core) {
+    lazyStreamDeck.core = (await import('./async-module')).default;
```

Comment:
It'd be nice to combine the reducer and API into a single lazy chunk instead of two